### DISCUSSION TITLE
Configuration wiki page was removed

### DISCUSSION THREAD

**Discussion by shamblonaut** [polybar/polybar#2863] [Q&A]
The [Configuration](https://github.com/polybar/polybar/wiki/Configuration) page on the polybar wiki was missing. Visiting that link redirected me to create a new file.

I have checked the wiki change history and came up on the cause of the problem, [someone just removed the entire file](https://github.com/polybar/polybar/wiki/_compare/5ebabf3349367b6494104e46eefd93a4a1a01d1c). I have restored the file now.

**Comment by patrick96** [comment]
Thanks!

That's the third seemingly malicious edit this week. And some of these changes don't show up on the GitHub dashboard or the RSS feed, so I don't even see them.

I will make the wiki read-only for the time being.

**Reply by Zinvoke** [reply]
I have been going around to multiple repositories that he has been targeting some are pretty big and letting them know he has tons of accounts though :/ everyone please report this account:
https://github.com/AhmedAboSamra/VideoCaptureUtility/releases/tag/42
this is where hes linking all of the wiki's malware links to as you can see he has uploaded multiple zips with diff names some of them which he is targeting and it was recently updated

**Reply by patrick96** [reply]
Reported. Thanks for all your efforts @Zinvoke
